<!DOCTYPE html>
<html>
	<head>
		    <meta charset="utf-8">
		<title>CVE-2020-15701 - Vulnerability Data</title>
		    <link rel="shortcut icon" type="image/png" href="/favicon.ico" />
	</head>
<body>
<h2>Vulnerability Data</h2>
<h1>CVE-2020-15701</h1>
<p>An unhandled exception in check_ignored() in apport/report.py can be exploited by a local attacker to cause a denial of service. If the mtime attribute is a string value in apport-ignore.xml, it will trigger an unhandled exception, resulting in a crash. Fixed in 2.20.1-0ubuntu2.24, 2.20.9-0ubuntu7.16, 2.20.11-0ubuntu27.6.</p>

<dl>
	<dt>Published</dt>
	<dd>2020-08-06</dd>

	<dt>Modified</dt>
	<dd>2020-09-14</dd>
</dl>

<h3>References</h3>
<ul>
	<li><a href="https://launchpad.net/bugs/1877023">CONFIRM:https://launchpad.net/bugs/1877023</a></li>
<li><a href="https://usn.ubuntu.com/4449-1">CONFIRM:https://usn.ubuntu.com/4449-1</a></li>
<li><a href="https://usn.ubuntu.com/4449-1/">UBUNTU:USN-4449-1</a></li>
<li><a href="https://usn.ubuntu.com/4449-2/">UBUNTU:USN-4449-2</a></li>
</ul>

<footer>
	<p><a href="https://reconmap.org">Track security vulnerabilities</a> with <strong>Reconmap</strong>.</p>
</footer>
